pkg/systemd/overview.jsx

```diff
@@ -39,10 +39,13 @@ import { UsageCard } from './overview-cards/usageCard.jsx';
 import { SuperuserAlert } from './superuser-alert.jsx';
 import { SuperuserIndicator } from "../shell/superuser.jsx";
 import { ShutdownModal } from 'cockpit-components-shutdown.jsx';
+import { WithDialogs, DialogsContext } from "dialogs.jsx";
 
 const _ = cockpit.gettext;
 
 class OverviewPage extends React.Component {
+    static contextType = DialogsContext;
```

Comment:
Note to self: https://reactjs.org/docs/context.html#classcontexttype

This looks weird at first sight, as it's not referenced anywhere. This ought to go into the docs section of pkg/lib/dialogs.jsx.